lib/icedfrisby.js

```diff
@@ -1151,6 +1162,22 @@ class Frisby {
     const describeFn = this._only ? describe.only : describe
 
     describeFn(self.current.describe, function () {
+      before(function () {
+        if (self.current.useApp) {
+          const { app, baseUri } = self.current.useApp
+          const { server, uri } = startApp(app, baseUri)
```

Comment:
I get the feeling that we shouldn't have to start the app here because we already called `startApp()` (and therefore `listen()`) above when `obj.useApp` is truthy. I'd have to take a closer look to confirm my thoughts though.

Reply:
That's true, though that line is in `globalSetup`. A user would either configure the app in `globalSetup` or in `frisby.create().useApp()`. They probably should not do it in both places.

Reply:
Gotcha, my mistake.